armi/physics/neutronics/settings.py

```diff
@@ -324,6 +329,13 @@ def defineSettings():
             description="Method to determine broad group cross sections for assemblies",
             options=["", "MC2v2", "MC2v3", "MC2v3-PARTISN", "SERPENT"],
         ),
+        setting.Setting(
+            CONF_LATTICE_PHYSICS_UPDATE_FREQUENCY,
+            default="BOC",
+            label="Frequency of lattice physics updates",
+            description="Define the frequency at which cross sections are updated with new lattice physics interactions.",
+            options=["", "BOL", "BOC", "everyNode", "firstCoupled", "all"],
```

Comment:
I would recommend pulling the options straight from the enum so that you don't have to keep this manually updated in the future.

Reply:
Good idea!